### ISSUE TITLE
Refactor block handling methods

### ISSUE THREAD

**Issue by vldm** [stegos/stegos#389]
Currently some part of blockchain state are located in NodeService. And methods `on_XX_block_registred` are called evetry time when new block is writed to blockchain, we should merge logic that is splited in this method, and move it with state to Blockchain module.

We need to move this logic into blockchain, in order to be able to recover state from disk.

Blocks: https://github.com/stegos/stegos/issues/8

**Comment by rtsisyk** [comment]
This ticket is not blocked anymore.

**Comment by vldm** [comment]
@rtsisyk Its already done in https://github.com/stegos/stegos/pull/395
